usaspending_api/references/management/commands/load_agencies.py

```diff
@@ -0,0 +1,43 @@
+from django.core.management.base import BaseCommand, CommandError
+from usaspending_api.references.models import Agency, SubtierAgency
```

Comment:
I guess this whole loader needs to be updated. I'm ok with this not working atm, though since it's not really in scope of the sprint.